packages/language-server/src/plugins/typescript/language-service.ts

```diff
@@ -294,6 +294,7 @@ async function createLanguageService(
 			noEmit: true,
 			declaration: false,
 
+			resolveJsonModule: true,
```

Comment:
This is part of our starter `tsconfig.json` and also is in our TypeScript docs. So a lot of users already had this, luckily. Now everyone will have it :smile: